### PR TITLE
Fix build in webpack

### PR DESCRIPTION
**Pull request by ciromattia**

### CODE REVIEW

**Comment by FezVrasta** [comment]
Thanks!

The CI build is broken, may you take a look?

**Comment by ciromattia** [comment]
Yes, I see, it's because node-sass doesn't like the ~ notation for the library, I'm going to look into it!

**Comment by FezVrasta** [comment]
I found this module that should add tidle imports support to node-sass.

https://www.npmjs.com/package/node-sass-tilde-importer

It's not a super clean solution probably but still better than asking people to manually define include paths in their own tooling...

**Comment by ciromattia** [comment]
Good catch, I think it's the way to go then!
(Maybe you'll want to squash the commits, I had some issues with the merge from upstream!)

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "build": "npm run build:js && npm run build:css",
     "build:js": "rollup js/index.js --output=dist/js/bootstrap-material-design.js -m --config=rollup.config.js && npm run build:js-min",
     "build:js-min": "minify dist/js/bootstrap-material-design.js --out-file dist/js/bootstrap-material-design.min.js",
-    "build:css": "mkdir -p dist/css && node-sass --include-path node_modules scss/bootstrap-material-design.scss > dist/css/bootstrap-material-design.css && npm run build:css-min",
+    "build:css": "mkdir -p dist/css && node-sass --include-path node_modules --importer=node_modules/node-sass-tilde-importer scss/bootstrap-material-design.scss > dist/css/bootstrap-material-design.css && npm run build:css-min",
```
Comment (FezVrasta) [inline package.json]: I don't think `--include-path` is necessary anymore?

**Comment by FezVrasta** [comment]
Just wondering, what about ruby-sass?

**Pull request closed without merging**